config/controller.yaml

```diff
@@ -118,6 +118,8 @@ spec:
           value: /etc/ssl/certs
         - name: METRICS_DOMAIN
           value: tekton.dev/pipeline
+        - name: KUBERNETES_MIN_VERSION
+          value: v1.23.0
```

Comment:
Now that `main` is on min version 1.23 already, this is not needed anymore.

Reply:
Removed 👼🏼